src/test/scala/nl/knaw/dans/easy/bagindex/ServerTestSupportFixture.scala

```diff
@@ -40,5 +40,5 @@ trait ServerTestSupportFixture {
     }
   }
 
-  def successful: (Int, String) = (200, "EASY Bag Index running.")
+  def successful(version: String): (Int, String) = (200, s"EASY Bag Index running v$version\n.") //TODO how does this newline end up in the response?
```

Comment:
Good catch! You need to add `.stripLineEnd` in `ConfigurationComponent`. See https://github.com/DANS-KNAW/easy-add-new-deposit-agreement/blob/ef68e9f74e092c87e4fca2459fca34c1310c2714/src/main/scala/nl/knaw/dans/easy/addnewagreement/Configuration.scala#L37 for an example.